File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1732,6 +1732,74 @@ document.Close(True)
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/HTML-to-PDF-rotate-page).
 
+## HTML Header and Footer
+
+You can set Html string as header and footer when converting HTML to PDF, utilize the **HtmlHeader** and **HtmlFooter** properties in the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class. Please refer to the following code example.
```

Comment:
When converting HTML to PDF, you have the option to set an HTML string as both the header and footer. To do this, make use of the HtmlHeader and HtmlFooter properties within the [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class.

Reply:
changed